test/fixtures/types/route-types.ts

```diff
@@ -53,6 +53,15 @@ export interface Routes {
       }
     }
   }
+  '/deprecated/undocumented/endpoint': {
+    route: '/deprecated/undocumented/endpoint'
+    method: 'GET'
+    queryParams: Record<string, unknown>
+    jsonBody: Record<string, unknown>
+    commonParams: Record<string, unknown>
+    formData: Record<string, unknown>
+    jsonResponse: {}
```

Comment:
I wonder if nextlove should generate this type as `Record<string, never>` instead. Anyway, ok to add an eslint disable next line here.

Reply:
a3224e785f8789d646b32c0b682a76437b47a7b3